docs/blog/2018-12-15-develop-and-deploy-your-own-react-monorepo-app/index.md

```diff
@@ -0,0 +1,598 @@
+
+The same goes for those who have worked as part of a large company/team. Branding guidelines come in the form of css style-guides or reusable components. Separating the discrete packages from what’s shared becomes a hassle if everything’s stored together. You may even have to deal with several similar projects thrown about in different repos, which can be an arse-ache to keep track of. These are some of the problems monorepos solve.
+
+![Monorepo Diagram](./images/monorepo-diagram.jpeg)
+
+A monorepo is an architectural design pattern that allows us to maintain several packages within a single repo. This reduces the complexity when dealing with packages that share in-house and third-party dependencies.
+
+Think of monorepos like a well-organised supermarket. Each aisle contains a particular kind of food, making it easy to navigate and find the item of food you want. Supermarkets also contain a variety of different foods so you don’t need to go to a series of specific shops to find your gluten-free, dairy-free banoffee pie (which is very much a [thing](https://glutenfreecuppatea.co.uk/2016/02/14/recipe-healthy-banoffee-pie-gluten-free-vegan-refined-sugar-free-dairy-free/)).
+
+With monorepos the code is split into specific packages (aisles). Meaning you can easily navigate the project to find the component or module you want to work on. It also means not needing to maintain separate repos when making changes that affect multiple parts of your stack (the single shop).
+
+Everything is accessible from a single place, while still being organised enough to navigate painlessly.
+Enough with the theory, let’s move on to the tutorial, which is split into 2 sections: development and deployment.
+
+Development — We’ll configure an existing application, built with Gastby, into a monorepo. If you haven’t heard of Gatsby before, then take some time to [read up on it](https://www.gatsbyjs.org/). I won’t delve too much into its implementation details because it’s beyond the scope of this article. We’ll attend the lavish party that the great Gatsby throws for us and we won’t ask why or how they did it. Another tool we’ll use is [Lerna](https://github.com/lerna/lerna), which will manage the dependencies of our project. Lerna will also allow us to link packages together which we’ll use to share components.
+
+Deployment — After we’ve converted our app into a monorepo we’ll hook our GitHub account to Travis-CI and create a CI pipeline. The CI pipeline will test, build and deploy our application. W̵e̵’̵l̵l̵ ̵b̵e̵ ̵u̵s̵i̵n̵g̵ ̵H̵e̵r̵o̵k̵u̵ ̵̵to h̵andle-d̵e̵p̵l̵o̵y̵m̵e̵n̵t̵s ̵t̵o̵o̵l̵ ̵b̵e̵c̵a̵u̵s̵e̵ ̵i̵t̵’̵s̵ ̵e̵a̵s̵y̵ ̵t̵o̵ ̵u̵s̵e̵,̵ ̵f̵l̵e̵x̵i̵b̵l̵e̵ ̵a̵n̵d̵ ̵i̵n̵t̵e̵g̵r̵a̵t̵e̵s̵ ̵r̵e̵a̵l̵l̵y̵ ̵w̵e̵l̵l̵ ̵w̵i̵t̵h̵ ̵T̵r̵a̵v̵i̵s̵-̵C̵I̵.̵ I changed my mind after spending three days trying to get my Monorepo deployed with Heroku, and managed it in an hour with Now. So we’re using Now!
```

Comment:
Please remove quotes. Btw, does this still work with Now v2?

Reply:
this issue gets solved by explicitly stating the version of Now being used from within the `now.json` file